CONTRIBUTING.md

```diff
@@ -5,29 +5,37 @@
 
 ## Getting Started
 
-To work on this project please create an account on Cloudflare and Discord.
-Also, create a personal discord server.
-Copy the guild id of your server and save it in a .env file as `DISCORD_GUILD_ID`
-
-To get the server ID for the first parameter, open Discord, go to Settings > Advanced and enable developer mode.
-Then, right-click on the server title and select "Copy ID" to get the guild ID.
-
+To contribute to this project, you have to set up few things first.
+
+- Create an account on Cloudfare Workers and Discord if you don't already have one
+- Create a personal discord server
+- Fork and clone this repo
+- Install volta - this will check if you have the right node version or not
+- Create a .env file
+
+Next, you need to gather few details that will be crucial for setting up your local environment. The IDs that you will gather in the next steps, will be stored as secret keys on the Cloudfare workers platform, where the application will be hosted.
+
+- To get the guild ID of your server :
+  - Open your discord server
+  - Click on the server name in the top-left corner
+  - Click on Server Settings
+  - Click on the Widget tab in the side panel
+  - Copy/paste the Server ID in the .env file as `DISCORD_GUILD_ID`
 - Visit [Discord Developer Portal](https://discord.com/developers/applications)
 - Click on new application.
+- Under the General Information panel
+  - copy/paste the Application ID in the .env file as `DISCORD_APPLICATION_ID`
+  - copy/paste the Public Key in the .env file as `DISCORD_PUBLIC_KEY`
 
-Gather the following details from the Developer portal and save it in a .env file.
+Next, you need to create your own bot. The idea here is that you will have your own bot that can be used to run all commands in your private server. You can add new features and test it there to make sure everything works as expected, before deploying the code on production.
 
-```
-DISCORD_TOKEN: Available in bot panel of your discord bot after clicking reset token button.
-DISCORD_APPLICATION_ID: Available in general panel of your discord bot.
-DISCORD_PUBLIC_KEY: Available in general panel of your discord bot.
-```
+- To create a bot, go to the Bot panel and click on 'Add Bot'
```

Comment:
Isn't it the direct applications panel?

Reply:
![add_bot_discord](https://user-images.githubusercontent.com/84833473/232298842-df7c9e48-7731-4068-98b0-6d914e9dfce7.PNG)

Reply:
you will find it in oauth 2 URL generator

Reply:
is it directly available when you have no bots? i don't think this interface opens up without any bot created

Reply:
@RitikJaiswal75  in any case, the 'bot' option is always available in the side panel